packages/calypso-e2e/src/rest-api-client.ts

```diff
@@ -360,7 +360,7 @@ export class RestAPIClient {
 			);
 		}
 
-		if ( response.errors === [] ) {
+		if ( response.errors.length === 0 ) {
 			console.log( response );
 			throw new Error( `Failed to create invite: ${ response.errors }` );
```

Comment:
I think we need to check if there _are_ errors here!

Reply:
Ah okay, I should have written some comments because it isn't straightforward at first glance. 

The first `hasOwnProperty` is checking for API errors.
The second `error` check is checking for errors relating to the invite itself. 

I've added comment, fixed the logic for the `if` clause and added a helpful `console.error` level output.